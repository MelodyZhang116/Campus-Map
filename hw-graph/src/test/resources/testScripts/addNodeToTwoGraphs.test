# test that add node to the graph will not create incorrect behavior to other graph.

# Create a graph
CreateGraph graph1

# Add a node
AddNode graph1 n1

# Print the nodes in the graph
ListNodes graph1

# Create another graph
CreateGraph graph2

# Add a node to the second graph
AddNode graph2 n2

# Print the nodes in the second graph
ListNodes graph2

# Print the nodes in the first graph again to make sure it is still correct
ListNodes graph1