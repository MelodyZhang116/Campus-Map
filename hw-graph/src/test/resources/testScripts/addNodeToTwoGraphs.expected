# test that add node to the graph will not create incorrect behavior to other graph.

# Create a graph
created graph graph1

# Add a node
added node n1 to graph1

# Print the nodes in the graph
graph1 contains: n1

# Create another graph
created graph graph2

# Add a node to the second graph
added node n2 to graph2

# Print the nodes in the second graph
graph2 contains: n2

# Print the nodes in the first graph again to make sure it is still correct
graph1 contains: n1